### PR TITLE
Danny/readonly crew section

### PR DESCRIPTION
**Pull request by odin5on**

### CODE REVIEW

File: desktop/src/scenes/Analyze/analyze.jsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ export class Analyze extends Component {
   }
 
   back = () => {
-    this.props.history.push(`/`)
+    window.history.back()
```
Comment (joshuawootonn) [inline desktop/src/scenes/Analyze/analyze.jsx:99]: Shouldn't this depend on whether you are in electron or not? There is an existing `isElectron` function you can use for this.

File: desktop/src/scenes/Analyze/project.jsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ export class Project extends Component {
   }
 
   back = () => {
-    this.props.history.push(`/`)
+    this.props.history.goBack()
```
Comment (joshuawootonn) [inline desktop/src/scenes/Analyze/project.jsx:52]: Same as `isElectron` comment

File: desktop/index.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
     <meta name="theme-color" content="#000000" />
     <link rel="manifest" href="/manifest.json" />
-    <link rel="shortcut icon" href="/favicon.ico" />
+    <link rel="shortcut icon" href="./assets/icon.ico" />
```
Comment (joshuawootonn) [inline desktop/index.html:10]: If this works on both desktop and web then great. I am a bit worried that it wont work on desktop.

At some point it looks like `favicon.ico` in the `/desktop/public` directory got renamed to `/desktop/public/icon.ico`. 

If ^ doesn't work for both versions then I would try `/icon.ico`

File: desktop/src/scenes/Analyze/crew.jsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+                  id: item.id,
+                  data: { ...item },
+                  name: item.name,
+                }
+              })}
+              className="text-slate-900 py-4 flex-grow"
+            />
+            <Tooltip title="Go Back" placement="bottom">
+              <IconButton
+                className="!hidden md:!block"
```
Comment (joshuawootonn) [inline desktop/src/scenes/Analyze/crew.jsx:106]: Kinda weird that you have to make these important. Are they both just being overwritten by material UI?

File: desktop/src/components/inputs/Day/day.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+    return (
+      <div className="text-black">
+        <input className="h-[36px] md:h-[38px] border border-gray-800 rounded" type="date" min={oneMonthAgo} max={today} value={value} onChange={(event) => onChange(event.target.value)} style={{ borderColor: 'rgb(204, 204, 204)' }} />
+      </div>
+    );
+  }
+}
+
+export default Day
```
Comment (joshuawootonn) [inline desktop/src/components/inputs/Day/day.jsx:19]: Make sure to add a new line here at the end.

**Comment by joshuawootonn** [comment]
I would prefer as much as possible for the net new files to be typescript.

**Comment by joshuawootonn** [comment]
Where does this take you once you have completed the form?

**Review by joshuawootonn — CHANGES_REQUESTED** [review]
You are on the way. I like that you are reusing `shiftCRUD.container.jsx` that is a good idea. Not sure this PR is all the way up to date or if you wanted a review yet, but just thought I would give feedback while I can.

File: desktop/src/scenes/Analyze/analyze.jsx (hunk 1 above)
Comment (odin5on) [reply desktop/src/scenes/Analyze/analyze.jsx:99]: tested this in the desktop app and it works fine

File: desktop/index.html (hunk 3 above)
Comment (odin5on) [reply desktop/index.html:10]: Works in both. I'm assuming the build just uses this for the app icon? In which case it works

**Comment by odin5on** [comment]
I started converting some to typescript, but I'd like to talk a little more about doing that with you

**Review by joshuawootonn — APPROVED** [review]

**Pull request merged**